internal/wfs/transaction.go

```
@@ -0,0 +1,57 @@
+package wfs
+
+import "encoding/xml"
+
+type Transaction struct {
+	XMLName        xml.Name       `xml:"http://www.opengis.net/wfs Transaction"`
+	Service        string         `xml:"service,attr"`
+	Version        string         `xml:"version,attr"`
+	XSI            string         `xml:",xmlns:xsi,attr"`
```

golangci-lint (1.64.8)

SA5008: invalid XML tag: unknown option "xmlns:xsi"

(staticcheck)